templates/azure/pro.html

```diff
@@ -385,7 +385,7 @@ <h4>Technical introduction to Ubuntu Pro</h4>
       </a>
       <h4>Simple VM provisioning with cloud-init</h4>
       <p>Cloud-init automates the provisioning of an operating system image into a fully running state. Typical tasks handled by cloud-init include networking and storage initialisation and optional package installation and configuration. This means users can build up-to-date images in no time and with minimum maintenance efforts.</p>
-      <p><a class="p-button--positive" href="https://assets.ubuntu.com/v1/05994c5c-Cloud-init_on_Azure.pdf">Download the e-book</a></p>
+      <p><a class="p-button--positive" href="engage/azure-cloud-init-whitepaper">Download the e-book</a></p>
```

Comment:
```suggestion
      <p><a class="p-button--positive" href="/engage/azure-cloud-init-whitepaper">Download the e-book</a></p>
```

Reply:
Sorry about that - it was clarifited mid PR. Cloud init should be gating. Scaling Azure is a simple download link.